R/install-github.r

```diff
@@ -110,6 +110,10 @@ remote_download.github_remote <- function(x, quiet = FALSE) {
     warning("GitHub repo contains submodules, may not function as expected!",
       call. = FALSE)
 
+  if (!quiet) {
```

Comment:
That's absolutely fine. 
Then this message (a few lines above) then needs to happen after "src" string is assembled.

  if (!quiet) {
    message("Downloading GitHub repo ", x$username, "/", x$repo, "@", x$ref)
  }

That should work.